app/layout.tsx

```diff
@@ -28,7 +28,7 @@ export default function RootLayout({
         <link rel="alternate icon" href="/SerotrackerLogo.png" />{" "}
         {/* Fallback for older browsers */}
       </head>
-      <body className={clsx(inter.className, "text-black no-scrollbar overflow-hidden")}>
```

Comment:
Initially added this in to avoid that large chunk of greenspace at the bottom, but then with out #id based navigation it broken the header and scroll.

So gonna leave for now as it is ugly but functionaly and create a ticket to resolve